q2_chemistree/_fingerprint.py

```diff
@@ -58,6 +59,8 @@ def fingerprint(sirius_path: str, features: MGFDirFmt, ppm_max: int,
     zodiac_threshold : threshold filter for molecular formula re-ranking.
                        Higher value recommended for
                        less false positives (float)
+    java_flags : str
+        Setup additional flags for the Java virtual machine.
 
     Raises
```

Comment:
should we add an exception when ```java_flags``` is an unrecognized string?

Reply:
Unfortunately it would be very hard to check the validity of the strings. Hence I think we should just let people specify whatever they might need. I've listed in the plugin help the parameters that you would usually want to change and what you would change those to.